docs/cron-workflows.md

```diff
@@ -109,6 +110,41 @@ For example, with `timezone: America/Los_Angeles`:
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+#### Skip forward (missing schedule)
+
+You can use `when` to schedule once per day, even if the time you want is in a daylight saving skip forward period where it would otherwise be scheduled twice.
+
+An example 02:30:00 schedule
+
+```yaml
+schedules:
+  - 30 2 * * *
+  - 0 3 * * *
+when: "{{= cronworkflow.lastScheduledTime == nil || (now() - cronworkflow.lastScheduledTime).Seconds() > 3600 }}"
+```
+
+The 3:00 run of the schedule will not be scheduled every day of the year except on the day when the clock leaps forward over 2:30.
+The `when` expression prevents this workflow from running more than once an hour.
+In that case the 3:00 run will run, as 2:30 was skipped over.
+
+!!! Warning "Can run at 3:00"
+ If you create this CronWorkflow between the desired time and 3:00 it will run at 3:00 as it has never run before.
```

Comment:
Since this was merged before I had a chance to review, can see what this syntax error currently looks like [live](https://argo-workflows.readthedocs.io/en/latest/cron-workflows/#skip-forward-missing-schedule):
![Screenshot 2024-10-17 at 8 02 07 PM](https://github.com/user-attachments/assets/2bcc3e0a-3b9a-45bc-bd96-5989b5b76365)